molo/surveys/templates/surveys/molo_survey_page.html

```diff
@@ -14,6 +14,7 @@ <h4 class="surveys__intro">{{ page.intro }}</h4>
     {% if form %}
       <form class="surveys__form" action="{% pageurl self %}{% if self.multi_step or self.has_skip_logic %}?p={{ fields_step.number|add:"1" }}{% endif %}" method="post">
         {% csrf_token %}
+        {{ form.media }}
```

Comment:
@tm-kn does this evaluate to an import script tag or does it drop the JS straight into the HTML?

Reply:
This evaluates to a <script> and <link> tags. https://docs.djangoproject.com/en/1.9/topics/forms/media/#extend

Reply:
@projectmushroom If we just wrap this in our tags to check whether it's freebasics and whether the phone fails - will this work?

Would you rather we added the JS inline?

Reply:
@nathanbegbie wrapping it in our FB check would be fine 👍 does the form need js to submit? the form would then be useless on freebasics?

Reply:
@projectmushroom No - the function of the JS is simply to count down the number of characters remaining while a user types their answer